src/category_theory/limits/shapes/constructions/binary_products.lean

```diff
@@ -0,0 +1,45 @@
+/-
+-- Copyright (c) 2020 Bhavik Mehta. All rights reserved.
+-- Released under Apache 2.0 license as described in the file LICENSE.
+-- Authors: Bhavik Mehta
```

Comment:
Usually we don't put the `-- ` at the beginning of the lines in the license block.
Pro tip: in VScode you can type `copy`-`<Tab>` and it will put the block in place. You only need to fill in your name.